cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/ZKConfigUtils.java

```diff
@@ -0,0 +1,103 @@
+        ZK_PROPERTIES.put(ExecutorConfig.ZOOKEEPER_SSL_TRUSTSTORE_TYPE_CONFIG, "zookeeper.ssl.trustStore.type");
+        ZK_PROPERTIES.put(ExecutorConfig.ZOOKEEPER_SSL_PROTOCOL_CONFIG, "zookeeper.ssl.protocol");
+        ZK_PROPERTIES.put(ExecutorConfig.ZOOKEEPER_SSL_ENABLED_PROTOCOLS_CONFIG, "zookeeper.ssl.enabledProtocols");
+        ZK_PROPERTIES.put(ExecutorConfig.ZOOKEEPER_SSL_CIPHER_SUITES_CONFIG, "zookeeper.ssl.ciphersuites");
+        ZK_PROPERTIES.put(ExecutorConfig.ZOOKEEPER_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG, "zookeeper.ssl.hostnameVerification");
+        ZK_PROPERTIES.put(ExecutorConfig.ZOOKEEPER_SSL_CRL_ENABLE_CONFIG, "zookeeper.ssl.crl");
+        ZK_PROPERTIES.put(ExecutorConfig.ZOOKEEPER_SSL_OCSP_ENABLE_CONFIG, "zookeeper.ssl.ocsp");
+    }
+
+    private ZKConfigUtils() {
+
+    }
+
+    /**
+     * Builds the ZooKeeper client configuration
+     * @param config The Cruise Control configuration
+     * @return ZKClientConfig for the ZooKeeper client
+     */
+    public static ZKClientConfig zkClientConfigFromKafkaConfig(KafkaCruiseControlConfig config) {
+        if (!config.getBoolean(ExecutorConfig.ZOOKEEPER_SSL_CLIENT_ENABLE_CONFIG)) {
+            return new ZKClientConfig();
+        } else {
+            ZKClientConfig clientConfig = new ZKClientConfig();
+            clientConfig.setProperty(ZKClientConfig.SECURE_CLIENT, "true");
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_CNXN_SOCKET_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_KEYSTORE_LOCATION_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_KEYSTORE_PASSWORD_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_KEYSTORE_TYPE_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_TRUSTSTORE_LOCATION_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_TRUSTSTORE_PASSWORD_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_TRUSTSTORE_TYPE_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_PROTOCOL_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_ENABLED_PROTOCOLS_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_CIPHER_SUITES_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_CRL_ENABLE_CONFIG, config);
+            setZooKeeperClientProperty(clientConfig, ExecutorConfig.ZOOKEEPER_SSL_OCSP_ENABLE_CONFIG, config);
```

Comment:
Can we simplify this using a loop over the keys of `ZK_PROPERTIES` and calling  `setZooKeeperClientProperty` within the loop? Presumably, this would help with code maintainability -- e.g. if we want to add one more config to `ZK_PROPERTIES` in the future.